src/zc/FileStorage/snapshotintime.py

```diff
@@ -47,37 +46,36 @@ def main(args=None):
         print(usage % sys.argv[0], file=sys.stderr)
         sys.exit(1)
 
-
     try:
         if len(args) > 2:
             inpath, stop, outpath = args
         else:
             inpath, stop = args
-            if inpath.endswith('.fs'):
-                outpath = inpath[:-3]+stop+'.fs'
+            if inpath.endswith(".fs"):
+                outpath = inpath[:-3] + stop + ".fs"
             else:
-                outpath = inpath+stop
+                outpath = inpath + stop
     except ValueError:
         print(usage % sys.argv[0], file=sys.stderr)
         sys.exit(1)
 
     if not os.path.exists(inpath):
-        print(inpath, 'Does not exist.', file=sys.stderr)
+        print(inpath, "Does not exist.", file=sys.stderr)
         sys.exit(1)
 
     try:
-        date, time = (stop.split('T')+[''])[:2]
-        year, month, day = map(int, date.split('-'))
+        date, time = (stop.split("T") + [""])[:2]
+        year, month, day = map(int, date.split("-"))
         if time:
-            hour, minute, second = (map(int, time.split(':'))+[0,0])[:3]
+            hour, minute, second = (map(int, time.split(":")) + [0, 0])[:3]
```

Comment:
Unrelated to this PR, but this is a Python 3 compatibility bug -- map() returns an iterator that does not support `__add__`.

This hints to me that this package is maybe unmaintained?